packages/build/webpack/webpack.base.js

```diff
@@ -23,7 +23,17 @@ const ReactRefreshPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
 const appDirectory = fs.realpathSync(process.cwd());
 const resolveApp = relativePath => path.resolve(appDirectory, relativePath);
 
-module.exports = function createConfig() {
+module.exports = function getWebpackConfigFactory() {
+  return {
```

Comment:
hm...if we decided to call it factory, let have it just provide factory methods for the rest? Here is an example https://gist.github.com/alex-kovoy/9a09f2f4bed18101d4bae061894aaf2c  Notice how json property became a function.

```ts
rules: [
  configFactory.rules.fonts(),
  configFactory.rules.svg(),
  configFactory.rules.images(),
  configFactory.rules.jsx(),
  configFactory.rules.css()]
```

Reply:
Done